pandas/core/generic.py

```diff
@@ -7315,18 +7316,18 @@ def describe(self, percentiles=None, include=None, exclude=None):
         Describing all columns of a ``DataFrame`` regardless of data type.
 
         >>> df.describe(include='all')
-                categorical  numeric object
-        count            3      3.0      3
-        unique           3      NaN      3
-        top              f      NaN      c
-        freq             1      NaN      1
-        mean           NaN      2.0    NaN
-        std            NaN      1.0    NaN
```

Comment:
is there a reason these are being changed? they are already in alphabetical order. I suppose you could supply columns on construction to guarantee the order.

Reply:
thanks for the suggestion. Updated it with your recommendation.